v0/destinations/ga/transform.js

```diff
@@ -725,9 +725,14 @@ function processSingleMessage(message, destination) {
   const messageType = message.type.toLowerCase();
   let customParams = {};
   let category;
-  let { enableServerSideIdentify, enhancedEcommerce } = destination.Config;
+  let {
+    enableServerSideIdentify,
+    enhancedEcommerce,
+    ecommerce
+  } = destination.Config;
   enableServerSideIdentify = enableServerSideIdentify || false;
   enhancedEcommerce = enhancedEcommerce || false;
+  ecommerce = ecommerce || false;
```

Comment:
let's make this true by default?
then we can support present production users.

Reply:
yes we should do that.

Reply:
thinking more, actually this is tricky.
making the default true here will lead to always ignoring the config. 
Is the webapp/config BE PR merged? We can make the default value true there (to support present production users)
And check if the config value is undefined in transformer (instead of present falsy check), then only make true (tp support present production users) else take it from config.
wdyt?

Reply:
I did not understand the 2 line.. 
yes they are merged but there I think the default value is false.